Server/mods/deathmatch/logic/packets/CEntityAddPacket.cpp

```diff
@@ -162,15 +162,12 @@ bool CEntityAddPacket::Write(NetBitStreamInterface& BitStream) const
             {
                 const char*         szName = iter->first.c_str();
                 const CLuaArgument* pArgument = &iter->second.Variable;
-                bool                bSynchronized = iter->second.bSynchronized;
+                ESyncType           syncType = iter->second.syncType;

```

Comment:
you're declaring syncType here but not using it. also if you're changing something here you probably want backwards compatible code https://github.com/multitheftauto/mtasa-blue/wiki/BitStream

Reply:
It's just a misprint. The PR is completely backward compatible.